examples/cassandra/cassandra-controller.yaml

```diff
@@ -26,7 +26,7 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
-          image: gcr.io/google_containers/cassandra:v6
+          image: gcr.io/google_containers/cassandra:v7
```

Comment:
The current `Dockerfile` copies `kubernetes-cassandra.jar` in, so I imagine @brendandburns updated this by rebuilding *that*, but I don't know what process he used for that, which sort of makes the whole Docker build process kind of irrelevant. :)